src/Components/DotCMS/NewsWidgets.js

```diff
@@ -24,10 +24,13 @@ class NewsWidget extends React.Component {
         };
     }
 
-    getNewsItems(pagination) {
+    async getNewsItems(pagination) {
+        const langId = await DotCMSApi.languages.getId(
```

Comment:
Is would be cleaner if instead of calling 2 methods in the same lib you can encapsulate all this in one method.

Reply:
https://github.com/dotCMS/dotcms-spa-edit-mode/blob/ef6f1d44382bbc72d25c998e63f8f1cac17cb235/src/App.js#L49

in here we only need the `getCode()` method, so that's why I splitted in 2